native/NativeSampleApps/MobileSyncExplorer/src/com/salesforce/samples/mobilesyncexplorer/ui/MainActivity.java

```diff
@@ -283,11 +309,11 @@ public boolean onQueryTextChange(String newText) {
 		return true;
     }
 
-	@Override
-	protected void onListItemClick(ListView l, View v, int position, long id) {
-		final ContactObject sObject = listAdapter.getItem(position);
-		launchDetailActivity(sObject.getObjectId(), sObject.getName(),
-				sObject.getTitle());
+	protected void onListItemClick(ContactObject contact) {
```

Comment:
This got a nice data-object-oriented change with the new `RecyclerView.Adapter`.